samples/vsphere/vcenter/setup/setup_cli.py

```diff
@@ -103,4 +104,8 @@ def build_arg_parser():
                         action='store',
                         help='NFS Server IP to setup datastore for samples run.'
                              'If not passed as argument, update testbed.py file')
+    parser.add_argument('-epassword', '--esxpassword',
```

Comment:
Moved after e2. Making short argument to -epass instead just to avoid ambiguous if we add -p as argument in future.
Added space.